packages/xo-web/src/xo-app/sr/tab-xostor.js

```diff
@@ -40,7 +42,14 @@ const RESOURCE_COLUMNS = [
   },
   {
     name: _('vdi'),
-    itemRenderer: ({ vdiId }) => vdiId !== '' && <Vdi id={vdiId} />,
+    itemRenderer: ({ vdiId }) =>
+      vdiId !== '' && (
+        <Copiable data={vdiId}>
+          <Tooltip content={_('copyUuid', { uuid: vdiId })}>
+            <Vdi id={vdiId} />
+          </Tooltip>
+        </Copiable>
```

Comment:
I thought there was a way to change the tooltip shown by `Copiable` but apparently it's not implemented, so nevermind, let's remove the `Tooltip` because it shows 2 tooltips. If we really want a custom tooltip, we'll have to implement it in `Copiable`.